web/src/sentry.ts

```diff
@@ -98,20 +87,20 @@ if (page_params.server_sentry_dsn) {
                 shouldCreateSpanForRequest,
             }),
         ],
-        allowUrls: url_matches,
         sampleRate: page_params.server_sentry_sample_rate ?? 0,
         tracesSampler(samplingContext) {
             const base_rate = page_params.server_sentry_trace_rate ?? 0;
             const name = samplingContext.transactionContext.name;
             return base_rate * (sample_rates.get(name) ?? 1);
         },
-        initialScope: {
-            tags: {
+        initialScope(scope) {
+            scope.setTags({
                 realm: sentry_key,
                 user_role: user_info.role ?? "Browser",
                 server_version: realm.zulip_version,
```

Comment:
Maybe `?? ZULIP_VERSION` here so we still have something if initialization of `realm` fails?

Reply:
If `realm` failed to initialize, it will be `realm` that’s undefined, not `realm.zulip_version`.

Reply:
Hmm, we probably still want `ZULIP_VERSION` if that fails.